app.py

```diff
@@ -7,6 +7,8 @@
 from db import query
 
 
+temp = query.get_all_employees()
```

Comment:
@algorithmike Once you make a pull request, you should remove any code you were only using for testing.